codes/zig/chapter_array_and_linkedlist/my_list.zig

```diff
@@ -1,132 +1,173 @@
 // File: my_list.zig
 // Created Time: 2023-01-08
-// Author: codingonion (coderonion@gmail.com)
+// Author: codingonion (coderonion@gmail.com), CreatorMetaSky (creator_meta_sky@163.com)
 
 const std = @import("std");
-const inc = @import("include");
+const utils = @import("utils");
 
 // 列表类
 pub fn MyList(comptime T: type) type {
     return struct {
         const Self = @This();
-        
-        arr: []T = undefined,                        // 数组（存储列表元素）
-        arrCapacity: usize = 10,                     // 列表容量
-        numSize: usize = 0,                           // 列表长度（当前元素数量）
-        extendRatio: usize = 2,                       // 每次列表扩容的倍数
-        mem_arena: ?std.heap.ArenaAllocator = null,
-        mem_allocator: std.mem.Allocator = undefined, // 内存分配器
+
+        items: []T, // 数组（存储列表元素）
```

Comment:
It’s recommended to use the `int` type rather than a generic type `T`. This book focuses on data structures, and we should avoid introducing programming-language design concepts so beginners don’t get confused.